tests/PositionTests.cpp

```diff
@@ -35,22 +35,25 @@ TEST_CASE("Null Move Test", "[Position]") {
 
 TEST_CASE("Hash Test", "[Position]") {
     Position pos{STARTPOS_FEN};
-
     Position::hash_type old_hash = pos.hash();
 
     pos.make_move({E2, E4});
     REQUIRE(pos.hash() != old_hash);
+    REQUIRE(pos.hash() == pos.calculate_hash());
     pos.unmake_move();
     REQUIRE(pos.hash() == old_hash);
 
     pos.make_move({B1, C3});
     REQUIRE(pos.hash() != old_hash);
+    REQUIRE(pos.hash() == pos.calculate_hash());
     pos.make_move({B8, C6});
-    REQUIRE(pos.hash() != old_hash);
```

Comment:
Don't remove this, the test is still correct

Reply:
done